packages/components/card/src/base-card/CardActions.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useCallback, useState } from 'react';
+import type { MouseEventHandler, ReactElement } from 'react';
+import { Button } from '@contentful/f36-button';
+import type { ButtonProps } from '@contentful/f36-button';
+import { MoreHorizontal } from '@contentful/f36-icons';
+import { Dropdown } from '@contentful/f36-components';
```

Comment:
shouldn't we wait till the Dropdown is migrated to be able to extract card to a separate package?

Reply:
I don't think we have to, but we can if you think that's better.